fast64_internal/f3d/f3d_gbi.py

```diff
@@ -2018,7 +2087,8 @@ def to_c(self):
         def spc(x):
             return "{" + ", ".join([str(a) for a in x]) + "}"
 
-        return "{{ " + ", ".join([spc(self.position), "0", spc(self.uv), spc(self.colorOrNormal)]) + " }}"
+        flag = "0" if self.packedNormal == 0 else "{0:#06x}".format(self.packedNormal)
```

Comment:
Took your suggestion mainly; the # means "add 0x", that's why it has to be 6 wide instead of 4.

Not going to change the 0 because that's for non-F3DEX3, and that parameter ("flag") is unused there so it's not "hexadecimal zero", it's just "no parameter".